superset/databases/commands/export.py

```diff
@@ -32,6 +32,23 @@
 logger = logging.getLogger(__name__)
 
 
+def parse_extra(extra_payload: str) -> Dict[str, Any]:
+    try:
+        extra = json.loads(extra_payload)
+    except json.decoder.JSONDecodeError:
+        logger.info("Unable to decode `extra` field: %s", extra_payload)
+        return {}
+
+    # Fix for DBs saved with an invalid ``schemas_allowed_for_csv_upload``
+    schemas_allowed_for_csv_upload = extra.get("schemas_allowed_for_csv_upload")
+    if isinstance(schemas_allowed_for_csv_upload, str):
+        extra["schemas_allowed_for_csv_upload"] = json.loads(
+            schemas_allowed_for_csv_upload
+        )
```

Comment:
This won't be needed after we apply the migration fixing the DB, but for now we need it.